### PR TITLE
 Made 'value' required field when adding collateral

### PR DESCRIPTION
**Pull request by MYKatz**

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

This message was posted by [GitMate.io](https://gitmate.io) with timestamp signature Thanks for the PR, It will be reviewed Shortly!:1eKedm:GDx3CeoHQWls8ceTTGxqfCAx62U

File: app/scripts/controllers/loanAccount/NewLoanAccAppController.js [hunk 1]
```diff
@@ -265,6 +265,7 @@
                     scope.formData.collateral = [];
                     for (var i in scope.collaterals) {
                         scope.formData.collateral.push({type: scope.collaterals[i].type, value: scope.collaterals[i].value, description: scope.collaterals[i].description});
+
```
Comment (gkrishnan724) [inline app/scripts/controllers/loanAccount/NewLoanAccAppController.js]: @MYKatz  Please remove this extra line, you can remove the commit. It is not needed. Hint: `do git rebase -i HEAD~2` and delete this commit by prefixing it with `d`. Ping me if you have any doubts

File: app/views/loans/addloancollateral.html [hunk 2]
```diff
@@ -23,10 +23,11 @@ <h4>{{ 'label.heading.addloancollateral' | translate }}</h4>
                     </div>
                 </div>
                 <div class="form-group">
-                    <label class="control-label col-sm-2" for="value">{{ 'label.input.value' | translate }}</label>
+                    <label class="control-label col-sm-2" for="value">{{ 'label.input.value' | translate }}<span
+                            class="required">*</span></label></label>
```
Comment (gkrishnan724) [inline app/views/loans/addloancollateral.html]: Please remove the extra `</label>`

File: app/views/loans/addloancollateral.html [hunk 3]
```diff
@@ -23,10 +23,11 @@ <h4>{{ 'label.heading.addloancollateral' | translate }}</h4>
...
 
                     <div class="col-sm-3">
-                        <input id="value" type="text" class="form-control" ng-model="formData.value">
+                        <input id="value" type="text" class="form-control" ng-model="formData.value" required>
```
Comment (gkrishnan724) [inline app/views/loans/addloancollateral.html]: @MYKatz Thanks, Good job. Please also add a `late-validate` attribute and a `form-validate` tag. we want to keep the UI consistent :). Hint: Refer to this PR: https://github.com/openMF/community-app/pull/2271/files

**Comment by gkrishnan724** [comment]
Also, change your commit message, make it something like `Fix #2541 - Value field should be mandatory in add collateral`

**Comment by mbj36** [comment]
Looking forward to get this in @MYKatz 

CC - @gkrishnan724

**Comment by gkrishnan724** [comment]
This has been merged, Close this PR please. @MYKatz @mbj36 

https://github.com/openMF/community-app/pull/2620

**Comment by mbj36** [comment]
Thanks @gkrishnan724

**Pull request closed without merging**